target/linux/lantiq/base-files/etc/board.d/01_leds

```diff
@@ -55,6 +55,9 @@ WBMR300)
 	ucidef_set_led_switch "lan3" "LAN3" "wbmr300:green:lan3" "switch0" "0x20"
 	ucidef_set_led_default "router" "router" "wbmr300:green:router" "1"
 	;;
+RS353)
+        ucidef_set_led_default "dsl" "dsl" "$(lantiq_get_dt_led_chosen dsl)" "0"
```

Comment:
This line doesn't make much sense to me. If the LED is enabled by the bootloader for some reason, add a ```default-state = "off"``` to rs353:greeen:dsl in the dts and mention this oddity in the commit message.

See: http://lxr.free-electrons.com/source/Documentation/devicetree/bindings/leds/leds-gpio.txt?v=4.4